packages/terra-date-picker/tests/nightwatch/date-picker-spec.js

```diff
@@ -68,5 +67,18 @@ module.exports = {
     browser.expect.element('.terra-DatePicker-range > .terra-DatePicker').to.be.present;
   },
 
+  'Displays the calendar button with a height that matches the input ': (browser) => {
+    browser.url(`http://localhost:${browser.globals.webpackDevServerPort}/#/tests/date-picker-tests/default`);
+
+    let inputHeight;
+
+    browser.getElementSize('.terra-DatePicker-input', function getHeight(result) {
```

Comment:
Why don't use arrow callback in here and disable prefer-arrow-callback at top of the file?